Lib/test/test_traceback.py

```diff
@@ -4743,9 +4751,9 @@ def bar():
                 e, capture_locals=True
             )
         lines = "".join(exc.format(colorize=True))
-        red = _colorize.ANSIColors.RED
-        boldr = _colorize.ANSIColors.BOLD_RED
-        reset = _colorize.ANSIColors.RESET
+        red = colors["e"]
+        boldr = colors["E"]
+        reset = colors["z"]
```

Comment:
Why the single letter? I find this quite unreadable (like if you just read it immediately I think 'what the hell is "e" here).

Reply:
We can modify tests later to be more to your liking. This one now follows what I did with syntax highlighting. The idea was for the color tag braces to be as small as possible to not overwhelm the text that they are tagging. In other words, for the reader to be still able to "look through" the tags and see the contents below. I'm not married to it, if you want to change it later, go ahead.